codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithyjava/generator/library/ToDafnyLibrary.java

```diff
@@ -191,7 +195,12 @@ MethodSpec opaqueError() {
       .returns(dafnyError)
       .addModifiers(PUBLIC_STATIC)
       .addParameter(opaqueError, VAR_INPUT)
-      .addStatement("return $T.create_Opaque($L.obj())", dafnyError, VAR_INPUT)
+      .addStatement(
+        "return $T.create_Opaque($L.obj(), dafny.DafnySequence.asString($L.alt_text()))",
```

Comment:
Technically, the same applies above.
If WE create the Opaque,
It should never be null.

But if a custom Keyring/CMM/CMC creates it, 
all hell breaks loose.